remotes/handlers.go

```diff
@@ -71,17 +71,17 @@ func MakeRefKey(ctx context.Context, desc ocispec.Descriptor) string {
 		}
 	}
 
-	switch mt := desc.MediaType; {
-	case mt == images.MediaTypeDockerSchema2Manifest || mt == ocispec.MediaTypeImageManifest:
+	switch {
```

Comment:
I dropped the `mt` alias here as it really only was an advantage when doing the inline string quality checks; now that we are using the helpers, `mt` adds more line noise in my opinion.

Having every line state which field of the descriptor is being tested makes this faster to read, in my opinion.